src/legacy/core_plugins/kibana/public/management/sections/objects/lib/find_objects.ts

```diff
@@ -18,13 +18,14 @@
  */
 
 import { kfetch } from 'ui/kfetch';
+import { SavedObjectsFindOptions } from 'src/core/public';
 import { keysToCamelCaseShallow } from './case_conversion';
 
-export async function findObjects(findOptions) {
+export async function findObjects(findOptions: SavedObjectsFindOptions) {
   const response = await kfetch({
     method: 'GET',
     pathname: '/api/kibana/management/saved_objects/_find',
-    query: findOptions,
+    query: findOptions as Record<string, any>,
```

Comment:
`KFetchQuery` type is not really useful 😕

Reply:
Yea. all `kfetch` usages will be removed when this moves to the NP plugin, so I did not really bother much about these force-casts.